src/main/java/org/example/movieSpring/model/Movie.java

```diff
@@ -0,0 +1,27 @@
+package org.example.movieSpring.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
these types of annotations are generally bad practice
assign getters and setters on fields where needed them not on whole classes

Reply:
ok